consensus/common/src/main/java/org/hyperledger/besu/consensus/common/bft/blockcreation/BftBlockCreator.java

```diff
@@ -83,6 +84,14 @@ private static MiningBeneficiaryCalculator miningBeneficiaryCalculator(
         forksSchedule.getFork(blockNum).getValue().getMiningBeneficiary().orElse(localAddress);
   }
 
+  private void setBlockPeriodSeconds(
```

Comment:
Moving this into the controller same way you did with Clique would be nicer

Reply:
done, please re-review